### PR TITLE
feat: support downloading assets from a specific release

### LINKED ISSUE #3: Downloading the assets of a specific release
Hello!

I know, the project description is "Download an asset from the **latest** GitHub release" but I wanted to ask nevertheless: are you thinking of adding an argument (e.g. `--release <TAG>`) for making it possible to download the assets of a specific release? If you're thinking of implementing this, I'd be happy to submit a PR about it. If not, feel free to close the issue 🙂 

Thank you! 🐻

### PR DESCRIPTION
**Pull request by orhun**
closes #3

rec

### CODE REVIEW

**Review by devmatteini — CHANGES_REQUESTED** [review]
Hi @orhun! Nice to see you back :smile: 
Thank you again for your contribution!

I left a couple of comments that I'd like you to check out.

Note:
Even if this feature is very small and doesn't change much code, I usually would have split this commit in two.
The first one would have changed the `github` module.
The second one would have changed the `cli` module.

**Comment by orhun** [comment]
Hey @devmatteini, thank you for your feedback! Let me know if it looks good now.

File: src/github/mod.rs [hunk 1]
```diff
@@ -13,11 +13,14 @@ pub struct Repository {
     pub repo: String,
 }
 
-pub fn latest_release(repository: &Repository) -> Result<Release, GithubError> {
+pub fn get_release(repository: &Repository, tag: Option<&Tag>) -> Result<Release, GithubError> {
```
Comment (devmatteini) [inline src/github/mod.rs:16]: You forgot to update all usages of `latest_release(repository)` to `get_release(repository, None)`.
If we don't do that the build is broken on this commit :)

**Review by devmatteini — CHANGES_REQUESTED** [review]

File: src/github/mod.rs (hunk 1 above)
Comment (orhun) [reply src/github/mod.rs:16]: Done, please review again.

**Review by devmatteini — APPROVED** [review]
All good now! Nice work @orhun!

**Pull request merged**